pkg/lmbench3/Fix-rpc.h-not-found.patch

```diff
@@ -0,0 +1,42 @@
+From a3423b73ee44631e7bb48f8d5967ca706e66567c Mon Sep 17 00:00:00 2001
+From: Hao Li <lihao2018.fnst@cn.fujitsu.com>
+Date: Mon, 14 Dec 2020 10:48:06 +0800
+Subject: [PATCH] Fix rpc.h not found
+
+Signed-off-by: Hao Li <lihao2018.fnst@cn.fujitsu.com>
+---
+ scripts/build | 2 +-
+ src/Makefile  | 4 ++++
```

Comment:
Thanks, is this rpc.h problem specific on Fedora or general on other os like debian, ubuntu? Want to confirm this "Fix-rpc.h-not-found.patch" doesn't impact other os when doing pack.

Reply:
This problem affects all distros which have libtirpc-devel package. I have verifed this file on all distros through pkgs.org. All distros will benefit from this PR.